graders/python/requirements.txt

```diff
@@ -1,28 +1,28 @@
 beautifulsoup4==4.12.3
-bokeh==3.4.1
+bokeh==3.4.2
 colormath==3.0.0
 defusedxml==0.7.1
-Faker==25.3.0
-folium==0.16.0
-ipython==8.25.0
+Faker==26.0.0
+folium==0.17.0
+ipython==8.26.0
 matplotlib==3.9.0
 nbconvert==7.16.4
 nbformat==5.10.4
 networkx==3.3
 nltk==3.8.1
 numpy==1.26.4
-openpyxl==3.1.3
+openpyxl==3.1.5
 pandas==2.2.2
 Pillow==10.3.0
 PuLP==2.8.0
 Pygments==2.18.0
 pyarrow==16.1.0
 pygraphviz==1.13
 requests==2.32.3
-scikit-image==0.23.2
+scikit-image==0.24.0
 scikit-learn==1.5.0
-scipy==1.13.1
+scipy==1.14.0
```

Comment:
Large package but seems to be mostly improvements: https://docs.scipy.org/doc/scipy/release/1.14.0-notes.html